### ISSUE TITLE
A11y_Model Builder_Next Steps_Name Role Value: Incorrect ControlType is defined for "Deploy Your Model" and "Improve the Model" links.

### ISSUE THREAD

**Issue by manaswita-chougule** [dotnet/machinelearning-modelbuilder#2683]
### GitHub Tags
#WCAG4.1.2; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Name Role Value; #Win32;

### Environment
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.2
Microsoft Windows 11 Enterprise Build 22621.1702

### Repro Steps

1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. 
7. Complete the required details of All the three(scenario, environment, data, Train, Evaluate, Consume) steps.
8. Next Steps screen will appear.
9. Tab till "Deploy Your Model" or "Improve the Model" tile.
10. Inspect the control.
11. Observe that incorrect ControlType is defined for "Deploy Your Model" and "Improve the Model" links.

### Actual Result
Incorrect ControlType as "Custom" is defined for "Deploy Your Model" and "Improve the Model" links. 

### Expected Result
Correct ControlType as "HyperLink" should be defined for "Deploy Your Model" and "Improve the Model" links. 

### User Impact
Screen reader users will not able to understand the control and it's functionality if the control type is incorrectly defined for the links.

### Attachment
![incorrect role - Next steps](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/d5ecb7b9-ff88-470b-bff9-006dc833de68)

Note Issue:
![code snippet_edit](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/9a09145c-fac8-4b15-9a69-727a8700604e)

**Comment by zewditu** [comment]
@manaswita-chougule , ready to validate by Model Builder version 16.17.0.2332602.

**Comment by zewditu** [comment]
@manaswita-chougule , ready to validate by Model Builder version 16.17.0.2332602.

**Comment by manaswita-chougule** [comment]
#Regressed:06-29-23; #Closed;
Verified the issue in below environment:
Visual Studio : Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.6.3
Model Builder : Version 17.17.0.2332602

The issue is fixed, hence closing the bug. Please find attachment below for reference. Thanks!
![Screenshot 2023-06-29 175313](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/b86d5799-5bb4-44ad-98cf-598af569aa09)

![Screenshot 2023-06-29 123700](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/93005912-b15e-4159-bbed-149647362171)
